corehq/apps/export/models/new.py

```diff
@@ -1006,6 +1009,15 @@ def _generate_schema_from_case_property_mapping(case_property_mapping, app_id, a
         return schema
 
     @staticmethod
+    def _generate_schema_for_parent_case(app_id, app_version):
```

Comment:
We should check for the existence of a parent case type here, and only add ExportGroupSchema if it exists (or at least add the "TODO" back :P)